app/authorisation.js

```diff
@@ -1,23 +1,21 @@
 var config = require('./config');
+const log = require('bole')('middleware');
 
 
-module.exports.denyUnauthorized = function (req, res, next) {
-
-  if (isAuthorized(req)) {
+module.exports = function (req, res, next) {
+  if (_isAuthorized(req)) {
     next();
-
   } else {
     res.sendStatus(403);
     next(new Error('User is not authorized'));
   }
 };
 
-function isAuthorized(req) {
+function _isAuthorized(req) {
```

Comment:
It doesn't make sense to prefix Javascript functions with `_` - that is a Python idiom indicating that a function or method is intended to be private.
All functions in Javascript are private to the module unless explicitly exported.

Reply:
Does this pass eslint?

Reply:
We don't have eslint running in this project. I'll add the `lint` task and see what I can savage.